### ISSUE TITLE
@jire is Platinum cheats using your source? a lot of features is same as Charlatano.

### ISSUE THREAD

**Issue by tamal777** [Jire/Charlatano#487]

**Comment by w-o-a-h** [comment]
you are retarded

**Comment by Jire** [comment]
Not as far as I know. It's likely the other way around 😉

**Comment by tamal777** [comment]
@jire lol. Your cheat is great no questions about that. Tomorrow I saw how platinum works and the human like aimbot is same as charlatano. OMG I thought is someone using your source to make money. I don't have the cheat, just saw a video.

**Comment by COLATHOS** [comment]
Fucking bastard the Platinium's coder if it is true !

**Comment by BBY203** [comment]
@Jire hey do you have some kinda access to the platinumcheats's source!? Can I have it too pls? Or did you just copied the idea not the source?

**Comment by BBY203** [comment]
@Jire hey do you have some kinda access to the platinumcheats's source!? Can I have it too pls? Or did you just copied the idea not the source?

**Comment by Jire** [comment]
No, the reason I said it's the other way around is because I made the Charlatano aimbot using InvokeStatic's UC thread as a reference to fully understand everything.

**Comment by BBY203** [comment]
Oh ok then.
